### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jasdips**
- Minor edits for consistency.
- No space before colon in the JSON example.
- Updated RDAP OpenID reference to its new RFC.

### CODE REVIEW

File: draft-regext-rdap-x-media-type.md [hunk 1]
```diff
@@ -113,43 +113,43 @@ they do not support, servers SHOULD NOT list extensions in the RDAP-X media type
...
 referenced in the href attribute, and some clients use this information to determine
-if the URI in the href attribute should be dereferenced.
+if the URI in the href attribute should be de-referenced.
 
 Servers MAY use the RDAP-X media type in the type attribute if a client
 has negotiated content with the server using the RDAP-X media type, 
 the resource referenced by the URI matches the RDAP-X media type, and
 the resource referenced by the URI is served by a server compliant with this specification.
-Otherwise, use of the `application/rdap+json` media type is RECOMMENDED when the URI
+Otherwise, use of the 'application/rdap+json' media type is RECOMMENDED when the URI
```
Comment (anewton1998) [inline draft-regext-rdap-x-media-type.md:152]: The backtick usage is intentional as it tell markdown to use code style formatting such as monospace fonts.

Such as application/rdap+json vs `application/rdap+json`

File: draft-regext-rdap-x-media-type.md [hunk 2]
```diff
@@ -244,12 +248,12 @@ consult [@!RFC9110] regarding caching and other uses of the `vary` header.
 
 [@?RFC6838, section 4.3] strongly discourages the creation of new parameters on existing
 media types to enable new features. As RDAP has always had extensions, it could be argued
-that adding an "extensions" parameter to the existing `application/rdap+json` media type
+that adding an "extensions" parameter to the existing 'application/rdap+json' media type
```
Comment (anewton1998) [inline draft-regext-rdap-x-media-type.md:251]: Same comment as above. Maybe we just double quote it. I don't know.

**Review by anewton1998 — COMMENTED** [review]
All my comments are "meh". Feel free to ignore them and merge this.

File: draft-regext-rdap-x-media-type.md (hunk 2 above)
Comment (anewton1998) [reply draft-regext-rdap-x-media-type.md:251]: the let us use double quotes.

Comment (jasdips) [reply draft-regext-rdap-x-media-type.md:251]: OK, let me do so.

**Pull request merged**